src/components/footer/footer.tsx

```diff
@@ -1,6 +1,6 @@
 'use client';
 
-import { Box, Button, Container, IconButton, Link } from '@mui/material';
+import { Box, Container, IconButton, Link } from '@mui/material';
```

Comment:
`Button` was an un-used import, and `npm run build` was failing on Vercel due to a failing ESLint rule.

Reply:
Interesting we didn't see that before.